### DISCUSSION TITLE
Changing the user with su doesn't work

### DISCUSSION THREAD

**Discussion by p13rr0m** [ghostty-org/ghostty#3695] [Q&A] [answered]
Changing the user on MacOS with `su - username` doesn't work.
 
- It doesn't source the shell configuration (e.g. .zshrc for zsh).
- Problems with backspace, tab completion and many other keys.

**Comment by 00-kat** [comment, marked as answer]
For 2: https://ghostty.org/docs/help/terminfo#sudo.
